AsyncDisplayKit/ASCellNode.mm

```diff
@@ -103,16 +103,16 @@ - (void)didLoad
   }
 }
 
-- (void)layout
+- (void)layoutSubnodes
 {
-  [super layout];
+  [super layoutSubnodes];
   
   _viewControllerNode.frame = self.bounds;
 }
 
-- (void)layoutDidFinish
+- (void)nodeDidLayoutSubnodes
 {
-  [super layoutDidFinish];
+  [super nodeDidLayoutSubnodes];
```

Comment:
@maicki @appleguy Should we consider eliminating this method? Couldn't they just override `layoutSubnodes` and call `[super layoutSubnodes]` then do whatever they want to do?

If we keep it, should we call it `didLayoutSubnodes`, since `node` is implied? 🙂